apps/camera/js/views/controls.js

```diff
@@ -157,6 +164,12 @@ module.exports = View.extend({
     debug('mode set pos: %s', this.switchPosition);
   },
 
+  setRecording: function(recording) {
```

Comment:
I'm not sure why this method needs to be introduced? It seems to just call the previous method. Perhaps not needed anymore?